packages/@ngtools/webpack/package.json

```diff
@@ -25,6 +25,7 @@
     "npm": ">= 3.0.0"
   },
   "dependencies": {
+    "@angular/service-worker": "1.0.0-beta.3",
```

Comment:
This shouldn't be here, since the webpack plugin itself doesnt use the service worker.